clvm_tools/cmds.py

```diff
@@ -148,7 +148,11 @@ def launch_tool(args, tool_name, default_stage=0):
         run_program = args.stage.run_program
 
     src_text = args.path_or_code
-    src_sexp = reader.read_ir(src_text)
+    try:
+        src_sexp = reader.read_ir(src_text)
+    except SyntaxError as ex:
+        print("FAIL: %s" % (ex))
+        return -1
```

Comment:
Why is this block necessary?

Reply:
yeah, it looks suspicious to swallow an error, and turn it into a return value

Reply:
Thanks - this is leftover debugging code.